utils/submissionDataFromGoogleMaps.ts

```diff
@@ -0,0 +1,124 @@
+import puppeteer from 'puppeteer'
+import dotenv from 'dotenv'
+
+//Load the api key for google maps
+dotenv.config()
+
```

Comment:
dotenv is not necessary anymore with the new node versions I think we can just use process.env.NAME_OF_ENV

Reply:
Oh cool. I didn't know that.

Reply:
I tried this. but for some reason it wasn't loading the variable and showing as undefined. Could be how I was using it but just in case I put it back for now